### PR TITLE
Update the AWS KMS secrets docs.

### PR DESCRIPTION
**Pull request by adityadani**

### CODE REVIEW

File: secrets/portworx-with-aws-kms.md [hunk 1]
```diff
@@ -113,45 +113,36 @@ If the CLI is used to authenticate with AWS, for every restart of PX container i
...
 
 ```
-$ pxctl volume create --secure --secret_key my_secret_key --size 20 encrypted_volume
+/opt/pwx/bin/pxctl secrets aws generate-kms-data-key --secret_id portworx_secret
+KMS Data Key successfully created.
 ```
+The above command generates a KMS Data Key and associates it with the ```portworx_secret```. For subsequent operations you can use the same ```portworx_secret```
 
-The above command generates a kms data key and associates it with the ```keyID``` my_secret_key. For subsequent operations you can use the same ```keyID```
+Run the following command to create an encrypted volume using the newly generate KMS Data Key.
```
Comment (piyush-nimbalkar) [inline secrets/portworx-with-aws-kms.md]: *generated

File: secrets/portworx-with-aws-kms.md [hunk 2]
```diff
@@ -161,13 +152,6 @@ A cluster wide secret key is a common key that can be used to encrypt all your v
...
-As an input to set-cluster-key command you have the following two options
-1. Provide the path to the pre-generated kms data key. PX will not
-generate a key if a valid path and a valid kms cipher blob text is
-provided.
-
-2. Provide a ```keyID```. If the ```keyID``` was previously used, the
-already generated kms data key will be set as the cluster wide secret
-key. If the ```keyID``` was never used before a new kms data key will
-be generated and set as the cluster wide secret key.
+As an input to the above command you can provide any ```secret_id``` that was used in the generate-kms-data-key command. From our example we can set the cluster secret key to ```portworx_secret``` . This command needs to be run just once for the cluster. If you have added the [cluster secret key through the *config.json*](#update-configjson), the above command will overwrite it. Even on subsequent Portworx restarts, the cluster secret key in *config.json* will be ignored for the one set through the CLI.
```
Comment (piyush-nimbalkar) [inline secrets/portworx-with-aws-kms.md]: nit: `generate-kms-data-key`
remove space after `portworx_secret`

Comment (piyush-nimbalkar) [inline secrets/portworx-with-aws-kms.md]: the link should have been #adding-aws-kms-credentials-to-configjson

**Comment by adityadani** [comment]
I have updated the PR.

**Pull request merged**